charts/fluent-bit/values.yaml

```diff
@@ -71,6 +71,8 @@ service:
   port: 2020
   labels: {}
   # nodePort: 30020
+  # loadBalancerIP:
+  # externalTrafficPolicy: Local
```

Comment:
```suggestion
  loadBalancerIP:
  externalTrafficPolicy:
```

Wherever possible values should be defined but left empty, this helps with automated tools. If you want to add an example you can add it as a comment below the value.